resources/public/api/conf/2.0/application.raml

```diff
@@ -43,7 +43,7 @@ documentation:
         /{taxYear}:
           uriParameters:
             taxYear:
-              description: 'The tax year to which the data applies. For example: 2022-23. The start year and end year must not span two tax years. The minimum tax year is 2022-23. No gaps are allowed, for example, 2022-24 is not valid. (The minimum tax year in Sandbox is 2021-22.)'
+              description: 'The tax year to which the data applies. For example: 2021-22. The start year and end year must not span two tax years. The minimum tax year is 2022-23. No gaps are allowed, for example, 2022-24 is not valid. (The minimum tax year in Sandbox is 2021-22.)'
```

Comment:
I think the description is still inaccurate here - it says "The minimum tax year is 2022-23" despite the spec saying the  MAXimum tax year is 2021-22.